src/target/windows/mod.rs

```diff
@@ -0,0 +1,61 @@
+use std::mem;
+use winapi::shared::minwindef::FALSE;
+use winapi::um::processthreadsapi::{
+    CreateProcessW, OpenProcess, PROCESS_INFORMATION, STARTUPINFOW,
+};
+use winapi::um::winbase;
+use winapi::um::winnt;
+
+/// This structure holds the state of the debuggee on windows systems.
+pub struct Target {
+    process_handle: winnt::HANDLE,
+}
+
+macro_rules! wide_string {
+    ($string:expr) => {{
+        use std::os::windows::ffi::OsStrExt;
+        let input = std::ffi::OsStr::new($string);
+        let vec: Vec<u16> = input.encode_wide().chain(Some(0)).collect();
+        vec
+    }};
+}
+
+impl Target {
+    /// Launch a new debuggee process.
+    pub fn launch(path: &str) -> Result<Target, Box<dyn std::error::Error>> {
+        let mut si: STARTUPINFOW = unsafe { mem::zeroed() };
+        let mut pi: PROCESS_INFORMATION = unsafe { mem::zeroed() };
```

Comment:
Nit: even though it's clear what these variables represent in the context, I think it's still better to use expanded names, e.g.
```suggestion
        let mut startup_info: STARTUPINFOW = unsafe { mem::zeroed() };
        let mut proc_info: PROCESS_INFORMATION = unsafe { mem::zeroed() };
```

Reply:
Done